File: @LV1/1.2/quadrant/quadrant.rb
def quadrant(x, y)
  if x > 0 and y > 0
    return 1 
  elsif x < 0 and y > 0
    return 2
  elsif x < 0 and y < 0
    return 3
  else 
    return 4 
  end
end

x = gets.chomp.to_i
y = gets.chomp.to_i
puts quadrant(x, y)

File: @LV1/1.2/quadrant/quadrant.hs
quadrant x y
    | x > 0, y > 0 = 1
    | x < 0, y > 0 = 2
    | x < 0, y < 0 = 3
    | x > 0, y < 0 = 4

main = do
    x <- readLn :: IO Int
    y <- readLn :: IO Int
    print $ quadrant x y